apps/ejabberd/src/mod_roster.erl

```diff
@@ -380,6 +388,11 @@ item_to_xml(Item) ->
 get_roster_by_jid_t(LUser, LServer, LJID) ->
     mod_roster_backend:get_roster_by_jid_t(LUser, LServer, LJID).
 
+get_roster_by_jid(LUser, LServer, LJID) ->
+    {atomic, Item} = transaction(LServer,
```

Comment:
Why exactly do we need transaction for only get if we don't modify the `Item` in this transaction?